models/model.js

```diff
@@ -13,3 +13,30 @@ exports.getReviewsData = () => {
       return reviews;
     });
 };
+
+exports.getReviewByIDData = (review_id) => {
+  return db
+    .query(`SELECT * FROM reviews WHERE review_id = $1`, [review_id])
+    .then(({ rows: review }) => {
+      if (review[0] === undefined) {
+        return Promise.reject({ status: 400, msg: "Bad Request" });
+      } else {
+        return review[0];
+      }
+    });
+};
+
+exports.getCommentsByIDData = (review_id) => {
+  return db
+    .query(
+      `SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.review_id FROM comments JOIN reviews ON reviews.review_id = comments.review_id WHERE reviews.review_id = $1 AND comments.review_id = $1 ORDER BY comments.created_at desc;`,
+      [review_id]
+    )
+    .then(({ rows: comments }) => {
```

Comment:
You will need to check whether the review exists as an empty array may occur for two reasons:

- review exists but has no comments.
- review does not exist.